File: CLASSROOM/KL_for/4_p.a.py
#obs: as duas progessoes nao funcionam para decrescentes negativas, devido a limitação do menor que limite.
from funcao import number, show, insert
def main():
    a_zero = number(insert('Digite o primeiro termo da PG: '))
    razao = number(insert('Digite o quociente da PG: '))
    limite = number(insert('Qual o limite da PG? '))
    progressao_geometrica(a_zero,razao,limite)
def progressao_geometrica(a_zero,quociente,teto):
    termo = a_zero 
    for i in range(a_zero,teto):
        
    while termo < teto:
        show(f'{termo}; ',end='')
        termo *= quociente
main()

File: KL_for/py/4_p.a.js
/*const readlineSync = require('readline-sync');*/
import { question } from './utils.js';

function main(){
    const a_zero = Number(question('Digite o primeiro termo da PG: '))
    const razao = Number(question('Digite o quociente da PG: '))
    const limite = Number(question('Qual o limite da PG? '))
    progressao_geometrica(a_zero,razao,limite);
}
function progressao_geometrica(a_zero,quociente,teto){
    show_inline('Termos: ')
    for(let i = a_zero; i < teto; i*=quociente){
        show_inline(`${i}; `)
    }
}
function show_inline(label){
    return process.stdout.write(label)

}
main()